src/theme/_content.scss

```
@@ -23,6 +23,18 @@
       letter-spacing: 1px;
       text-transform: uppercase;
     }
+
+    @container (min-width: #{$largest-mobile-screen} ) {
```

```suggestion
    @container (min-width: #{$tablet-breakpoint} ) {
```

(tablet-breakpoint is one pixel larger than largest-mobile-screen; the difference is important so that we don't end up with a divider but no date at exactly 768 pixels)